agent/command_test.go

```diff
@@ -168,3 +195,56 @@ func TestEndTaskSyncCommands(t *testing.T) {
 		})
 	}
 }
+
+func (s *CommandSuite) TestVarsAreUnsetAfterRunning() {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	var err error
```

Comment:
nit: move this to L232 so it's defined closer to where it actually gets initialized.